bytes.hh

```diff
@@ -66,6 +66,17 @@ sstring to_hex(bytes_view b);
 sstring to_hex(const bytes& b);
 sstring to_hex(const bytes_opt& b);
 
+template <typename T>
+inline
+bytes
+write_bytes_be(T datum) noexcept {
```

Comment:
This function makes sense only for integer types, right? A concept would be in order then (std::is_integral?).

Reply:
Also, why move this function into bytes.hh? This weird function is only used in token.hh, so I think it should be there. If we ever need it anywhere else (I hope we don't!) we can move it then, and probably will need to add a comment what it does.